index.html

```diff
@@ -0,0 +1,382 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Bootstrap demo</title>
+    <link rel="stylesheet" href="bs/bootstrap-5.3.0-dist/css/bootstrap.css" />
+    <link rel="stylesheet" href="bs/bootstrap-5.3.0-dist/js/bootstrap.js" />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <nav class="navbar navbar-expand-lg bg-body-tertiary">
+      <div class="container">
+        <a class="navbar-brand" href="#"><span>My Logo</span></a>
+        <button
+          class="navbar-toggler"
+          type="button"
+          data-bs-toggle="collapse"
+          data-bs-target="#navbarSupportedContent"
+          aria-controls="navbarSupportedContent"
+          aria-expanded="false"
+          aria-label="Toggle navigation"
+        >
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+            <li class="nav-item">
+              <a class="nav-link" href="#">Portfolil</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">About</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Contact</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <section class="portofolio">
+      <h1>
+        I’m Alexandra.<br />
+        Glad to see you!
+      </h1>
+      <p>
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <h3>LET’S CONNECT</h3>
+      <ul class="nav connect">
+        <li class="nav-item">
+          <a href="#"><img src="images/twitter.png" alt="" /></a>
+        </li>
+        <li class="nav-item">
+          <a href="#"><img src="images/linkin.png" alt="" /></a>
+        </li>
+        <li class="nav-item">
+          <a href="#"><img src="images/M.png" alt="" /></a>
+        </li>
+        <li class="nav-item">
+          <a href="#"><img src="images/git.png" alt="" /></a>
+        </li>
+        <li class="nav-item">
+          <a href="#"><img src="images/fingers.svg" alt="" /></a>
+        </li>
+      </ul>
+    </section>
+
+    <!--Worksection-->
+    <div class="card" style="max-width: 90%">
+      <div class="row g-0 work">
+        <div class="col-md-6">
+          <img src="images/one.png" class="img-fluid rounded-start" alt="..." />
```

Comment:
:tada: Great work with this activity. I think that adding a descriptive alt text to your img tag will enhance accessibility. I think you should pay attention to that.